server/asset/eth/eth.go

```diff
@@ -335,11 +334,11 @@ func (eth *Backend) Redemption(redeemCoinID, contractCoinID []byte) (asset.Coin,
 
 // ValidateCoinID attempts to decode the coinID.
 func (eth *Backend) ValidateCoinID(coinID []byte) (string, error) {
-	coinId, err := dexeth.DecodeCoinID(coinID)
+	txHash, err := decodeCoinID(coinID)
```

Comment:
This is going to OK because with https://github.com/decred/dcrdex/pull/1293, `ValidateCoinID` is not used for order funding with account based assets, only UTXO assets?  I think we need to document this here.